plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties

```diff
@@ -104,7 +104,7 @@ dataTransfer.producer.stream.processor.csv.property.header.description
 dataTransfer.producer.stream.processor.csv.property.header.name                 = Posi\u00E7\u00E3o do cabe\u00E7alho
 dataTransfer.producer.stream.processor.csv.property.nullString.description      = String pura  usado parar marca valor NULL
 dataTransfer.producer.stream.processor.csv.property.nullString.name             = Marcar valor NULL
-dataTransfer.producer.stream.processor.csv.property.quoteChar.description       = Carastere usado para citar valor 
+dataTransfer.producer.stream.processor.csv.property.quoteChar.description       = Caractere usado para citar valor 
 dataTransfer.producer.stream.processor.csv.property.quoteChar.name              = Caractere de cita\u00E7\u00E3o
 dataTransfer.producer.stream.processor.csv.property.timestampFormat.description = Formato padr\u00E3o Data/Hora 
 dataTransfer.producer.stream.processor.csv.property.timestampFormat.name        = Formato Date/time 
```

Comment:
`dataTransfer.producer.stream.processor.csv.property.timestampFormat.name = Formato Date/time`

This one also looks partly translated.